end-to-end-tests/tst/helpers/resources.py

```diff
@@ -0,0 +1,129 @@
+from resource_types import ResourceTypes
+from resource_status import ResourceStatus
+
+import re
+
+
+class Resources:
```

Comment:
Can we add docstrings to all public functions? It will make it easier to understand everything.

Reply:
Good suggestion! added docstrings to all functions and classes